package.json

```diff
@@ -32,138 +32,131 @@
-    "@angular/platform-browser": "5.1.3",
-    "@angular/platform-browser-dynamic": "5.1.3",
-    "@angular/upgrade": "5.1.3",
+    "@angular/common": "5.2.4",
+    "@angular/compiler": "5.2.4",
+    "@angular/core": "5.2.4",
+    "@angular/platform-browser": "5.2.4",
+    "@angular/platform-browser-dynamic": "5.2.4",
+    "@angular/upgrade": "5.2.4",
     "@manageiq/font-fabulous": "1.0.0",
-    "@manageiq/ui-components": "1.0.10",
-    "@uirouter/angularjs": "1.0.12",
+    "@manageiq/ui-components": "1.1.0",
+    "@uirouter/angularjs": "1.0.14",
     "actioncable": "5.1.4",
-    "angular": "1.6.8",
-    "angular-animate": "1.6.8",
+    "angular": "1.6.9",
+    "angular-animate": "1.6.9",
     "angular-base64": "2.0.5",
     "angular-bootstrap-switch": "0.5.2",
-    "angular-cookies": "1.6.8",
-    "angular-datatables": "0.6.2",
+    "angular-cookies": "1.6.9",
     "angular-drag-and-drop-lists": "2.1.0",
     "angular-dragdrop": "1.0.13",
-    "angular-file-saver": "1.1.3",
     "angular-gettext": "2.3.10",
     "angular-gettext-cli": "1.2.0",
-    "angular-messages": "1.6.8",
-    "angular-patternfly": "4.14.2",
-    "angular-resource": "1.6.8",
-    "angular-sanitize": "1.6.8",
-    "angular-smart-table": "2.1.11",
+    "angular-messages": "1.6.9",
+    "angular-patternfly": "4.14.9",
+    "angular-resource": "1.6.9",
+    "angular-sanitize": "1.6.9",
     "angular-ui-bootstrap": "2.5.6",
     "angular-ui-sortable": "0.18.0",
+    "autoprefixer": "6.7.7",
     "babel-core": "6.26.0",
     "babel-eslint": "8.2.1",
     "babel-loader": "7.1.2",
     "babel-preset-env": "1.6.1",
+    "body-parser": "1.18.2",
+    "bootstrap": "3.1.1",
     "bootstrap-combobox": "1.0.2",
+    "bootstrap-datepicker": "1.7.1",
     "bootstrap-switch": "3.3.4",
-    "brace": "0.11.0",
     "c3": "0.4.18",
-    "chalk": "2.3.0",
-    "clean-webpack-plugin": "0.1.17",
-    "copy-webpack-plugin": "4.3.1",
+    "clean-webpack-plugin": "0.1.18",
+    "copy-webpack-plugin": "4.4.1",
     "core-js": "2.5.3",
-    "css-loader": "0.28.8",
+    "css-loader": "0.28.9",
     "d3": "3.5.17",
-    "datatables.net-dt": "1.10.16",
-    "datatables.net-select-dt": "1.2.4",
     "ejs": "2.5.7",
     "es6-shim": "0.35.3",
     "eslint-config-angular": "0.5.0",
-    "eslint-plugin-angular": "3.1.1",
+    "eslint-plugin-angular": "3.2.0",
     "extract-text-webpack-plugin": "3.0.2",
     "file-loader": "1.1.6",
     "font-awesome": "4.7.0",
-    "glob": "7.1.2",
-    "html-loader": "0.5.4",
+    "html-loader": "0.5.5",
     "html-webpack-plugin": "2.30.1",
     "imports-loader": "0.7.1",
-    "jquery": "3.2.1",
+    "jquery": "3.3.1",
     "jquery-ui-bundle": "1.12.1",
-    "lodash": "4.17.4",
+    "lodash": "4.17.5",
     "merge": "1.2.0",
+    "moment": "2.20.1",
     "moment-timezone": "0.5.14",
     "ng-annotate-loader": "0.6.1",
     "ng-redux": "3.5.2",
-    "ngprogress": "VictorBjelkholm/ngProgress#v1.1.3",
+    "ngprogress": "VictorBjelkholm/ngProgress#07f8fab",
```

Comment:
This a `noVNC` look surprising - any reason to depend on a particular commit here?